pkg/apis/cr/v1alpha1/types.go

```diff
@@ -290,6 +290,7 @@ type KopiaServerSecret struct {
 	Hostname       string                `json:"hostname,omitempty"`
 	UserPassphrase *KopiaServerSecretRef `json:"userPassphrase,omitempty"`
 	TLSCert        *KopiaServerSecretRef `json:"tlsCert,omitempty"`
+	Options        map[string]int        `json:"options,omitempty"`
```

Comment:
Should we say `CacheOptions` or `ConnectOptions`?

Reply:
@pavannd1 I intentionally kept if generic so that in future if we have to pass other options for different actions, we can use the same map